modules/retail/pages/retail-deploy-terminals.adoc

```diff
@@ -0,0 +1,136 @@
+However, the {susemgr} database does not have this information to begin with.
+In order to tell {susemgr} what image to deploy on each terminal, you can set hardware type groups.
+After you have created your new hardware type group, you can apply the [systemitem]``saltboot-formula`` to the group and configure it for your environment.
+
+Hardware types allow you to group devices according to manufacturer and device name, so that all devices of a particular type can be managed as one.
+
+You will need to create appropriate hardware types in your system.
+
+Empty profiles can be assigned to a hardware type group either before or after registration.
+If an empty profile is not assigned to a hardware type group before registration, it will be assigned to group that best matches the product information available to it.
+
+For this procedure, you will require the system manufacturer name and product name for your terminal.
+
+.Procedure: Creating a Hardware Type group
+
+. Determine the hardware type group name.
+Prefix the name with [systemitem]``HWTYPE:``, followed by the system manufacturer name and product name, separated by a hyphen.
+For example:
++
+----
+HWTYPE:POSVendor-Terminal1
+----
++
+[IMPORTANT]
+====
+Only use colons (``:``), hyphens (``-``) or underscores (``_``) in hardware type group names.
+Spaces and other non-alphanumeric characters will be removed when the name is processed.
+====
++
+. In the {susemgr} {webui}, navigate to menu:Systems[System Groups], and click the btn:[Create Group] button.
+. In the [guimenu]``Create System Group`` dialog, create a new system group, using the hardware type group name you created earlier.
+
+
+== Assign and Configure the Saltboot formula for each hardware type group
+
+Each hardware type group must have the Saltboot formula applied.
+
+.Procedure: Assigning the Saltboot formula
+
+. Open the details page for your new hardware type group, and navigate to the [guimenu]``Formulas`` tab.
+. Select the [systemitem]``saltboot-formula`` and click btn:[Save].
+. Navigate to the new menu:Formulas[Saltboot] tab.
+. Configure the Saltboot formula according to the instructions in xref:retail:retail-formulas.adoc[].
+
+
+== Synchronize images to the branch server
+
+.Procedure: Synchronizing images to the branch server
+
+. On the {susemgr} server, run this command:
++
+----
+salt <branch_server_salt_id> state.apply image-sync
+----
+
+== Deploy images to the terminals
+
+.Procedure: Deploying images to the terminals
+
+. Power on your POS terminals.
+. The branch server will bootstrap the terminals according to the data you have provided.
+
+
+== Terminals based on {slsa}{nbsp}11 SP{nbsp}3
+
+POS Terminals based on {sls}{nbsp}11 SP{nbsp}3 can be deployed in much the same way as other terminals, with a few differences.
+
+* You must use the {slsa}{nbsp}11 template
+* {slsa}{nbsp}11 images need to be activated with the [systemitem]``SLES11SP4 i586`` and [systemitem]``SLES11SP4 SDK i586`` channels
```

Comment:
I am confused about this. Are you telling that SLE11SP3 images should be using channels from SLE11SP4???

Reply:
Copied from here: https://github.com/SUSE/spacewalk/issues/7680#issuecomment-489602534

Reply:
Ping Jiri and Ondrej, I think something is not correct :(

Reply:
In theory SLEPOS11SP3 should be used, as SLE11SP3 is EoL

Reply:
SP3 is EOL for general use (unless you have any support contract going even beyond LTSS), but it is still supported for the POS images until 2022.

Reply:
After confirmation with Joe:
Host for building images: SLES11-SP4 + SLEPOS 11-SP3, either 32-bit or 64-bit
Repositories for building the Terminal image: SLES11-SP3 + SLEPOS11-SP3, 32-bit only

Reply:
I have updated https://github.com/SUSE/spacewalk/issues/7680#issuecomment-489602534 according to this.